architecture-building-systems/CityEnergyAnalyst
# I1099 change temperature outputs in sc total and pvt total

changes in this PR:
- In `solar_collector.py` and `photovoltaic_thermal.py`
- New outputs in `SC_total.csv` and `PVT_total.csv`: The temperatures are average instead of sum now.
- Read inputs from `cea.config`
- Delete `settings.py`

to test:

- run `solar_collector.py` and `photovoltaic_thermal.py` and check `SC_total.csv` and `PVT_total.csv`
